bool ask_ldb             #To trigger false incase connection incative
---
string station_name
string connection_status #To show connection status. Helps to refer rdb if connection fails continuously